apps/staking/src/components/Layout.tsx

```diff
@@ -51,22 +51,52 @@ export const headerLinks = [
 ];
 
 export const footerLinks = [
-    {
-        label: 'Audit Report',
-        href: 'https://github.com/cartesi/pos-dlib/raw/develop/Smart%20Contract%20Security%20Audit%20Report%20-%20Staking.pdf',
-    },
     {
         label: 'CTSI Reserve Mining',
         href: 'https://cartesi.io/en/mine/',
     },
+    {
+        label: 'Audit Report',
+        href: 'https://github.com/cartesi/pos-dlib/raw/develop/Smart%20Contract%20Security%20Audit%20Report%20-%20Staking.pdf',
+    },
     {
         label: 'How to Run a Node',
         href: 'https://medium.com/cartesi/running-a-node-and-staking-42523863970e',
     },
+];
+
+export const footerSupport = [
+    {
+        label: `What's New`,
+        href: 'https://cartesi.io/blog/',
+    },
+    {
+        label: 'Support on Discord',
+        href: 'https://discord.com/invite/pfXMwXDDfW',
+    },
     {
         label: 'FAQ',
         href: 'https://github.com/cartesi/noether/wiki/FAQ',
     },
+    {
+        label: 'User Idea',
+        href: 'https://governance.cartesi.io/',
+    },
+    {
+        label: 'Video Guide',
```

Comment:
@dandheedge, 
I would remove that and keep the `how to run a node`. It is the same link. 

For the `User Idea`, I would replace the copy with `Governance`. The **user idea** was a feature discussed in the past related to Explorer Staking, but we did not proceed with that. If someone wants a feature it should be requested through GitHub for consideration.